src/gsy_e/gsy_e_core/simulation.py

```diff
@@ -496,7 +496,7 @@ def __init__(self, setup_module_name: str, simulation_config: SimulationConfig,
                  no_export: bool = False, export_path: str = None,
                  export_subdir: str = None, redis_job_id=None, enable_bc=False,
                  slot_length_realtime: Duration = None, incremental: bool = False):
-        self._status = SimulationStatusManager(
+        self.status = SimulationStatusManager(
```

Comment:
I assume that this was needed only in order to be accessed from the tests ? Reason I am asking is because I cannot see where this might be required, since the status is self-contained in the Simulation classes.